src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -925,7 +925,12 @@ private static ISymbolModel SetupDefaultNameSymbol(string sourceName)
   ""description"": ""The default name symbol"",
   ""datatype"": ""string"",
   ""forms"": {
-    ""global"": [ ""identity"", ""safe_name"", ""lower_safe_name"", ""safe_namespace"", ""lower_safe_namespace""]
+    ""global"": [ """ + IdentityValueForm.FormName
```

Comment:
I went back & forth on how to best format this too. Offline we decided to keep as is.